azure/templates/marketplace-stack-ha/mainTemplate.json

```diff
@@ -2,702 +2,736 @@
   "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "location": {
```

Comment:
The diff in this file seems very large comparing to marketplace-stack-management/mainTemplate.json
I think the indentation is different (2/4 spaces for the tabs).
Can you check it? 
It will be easier to look at the diff.

Reply:
updated indentation to match.